src/ansys/grantami/bomanalytics/_item_definitions.py

```diff
@@ -385,15 +385,21 @@ class CoatingReferenceWithIdentifier(IdentifierMixin, CoatingReference):
 class ProcessReference(RecordReference):
     # Because of ProcessSummaryResult, this is publicly documented.
     # Extends RecordReference without changes, to re-define the class name, because it appears in the repr.
-    """Represents a reference to a Process record."""
+    """.. versionadded:: 2.0
+
+    Represents a reference to a Process record.
+    """
```

Comment:
I think I'd prefer if we kept the docstring first, it's a lot easier to read when looking at the source. Would it look decent in docs if the directive was at the end of the docstring?

Reply:
Updated the docstring to move the directives to the bottom

Reply:
![image](https://github.com/ansys/grantami-bomanalytics/assets/6451062/ea61e546-e211-4184-9c7c-681dda449c39)